docs/maui/includes/validation-behavior.md

```diff
@@ -0,0 +1,13 @@
+### Common Validation Properties
```

Comment:
What do you think about specifying that these properties come specifically from `ValidationBehavior`?

```suggestion
### ValidationBehavior Properties

The following properties are implemented in `public abstract class ValidationBehavior`:
```

Reply:
Yeah that works well in case other decide to implement their own it offers a slight pointer